WireUI/Sources/WireIndividualToTeamMigrationUI/Protocols/AccountMigrationAnalyticsTrackerProtocol.swift

```diff
@@ -0,0 +1,54 @@
+//
+// Wire
+// Copyright (C) 2025 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+public protocol AccountMigrationAnalyticsTrackerProtocol {
```

Comment:
In my opinion analytics events should actually be something internal to the module where they are fired.
However, we decided to have all analytics events declare at a central place (WireAnalytics package/target). So I introduced this protocol in order to achieve a loose coupling between the `WireAuthentication` and `WireAnalytics`.